setup.cfg

```diff
@@ -28,3 +28,7 @@ match = (?!test_|fixes).*\.py
 add-ignore = D100,D107,D413
 add-select = D214,D215,D404,D405,D406,D407,D408,D409,D410,D411
 ignore-decorators = ^(copy_.*_doc_to_|on_trait_change|cached_property|deprecated|property|.*setter).*
+
+[pytest]
```

Comment:
Instead of putting these in a new (deprecated name) `[pytest]` section, these should go in the existing preferred `[tool:pytest]` section